config/routes.rb

```diff
@@ -172,16 +169,34 @@
     get :sample_locations, on: :collection
   end
 
-  authenticate :user, ->(u) { u.admin? } do
-    mount RESQUE_SERVER, at: "/resque"
-  end
+  # TODO: - David - Check how to do that without devise
```

Comment:
nit: could you link to a JIRA as well?

Reply:
Just realized that I left all these TODOs behind by mistake. I'm reviewing them all.